.circleci/config.yml

```diff
@@ -0,0 +1,148 @@
+# CircleCI 2.0 configuration file for running Python on macOS.
+#
+# Email notifications are configured only from web.
+# See "Email Preferences by Organization" in
+# https://circleci.com/account/notifications
+#
+# Based on these v1.0 steps:
+# https://pewpewthespells.com/blog/building_python_on_circleci.html
+#
+# This uses the v2 workflows and a separate commit status is sent for each
+# job.
+#
+# CircleCI 2.0 automatically sets a $BASH_ENV variable to a
+# random name in /tmp, and will source this file for each step.
+#
+version: 2
+
+jobs:
+  static_checkers:
+    working_directory: ~/static-checkers
+    docker:
+      - image: circleci/python:2.7
+    steps:
+      # Get the source.
+      - checkout
+
+      - run:
+          name: Prepare the environment.
+          command: |
+            python --version
+            pip install -q --user --ignore-installed --upgrade virtualenv
+            pip install -q tox --user
+            echo 'export PATH=~/.local/bin:$PATH' >> $BASH_ENV
+
+      # Run each checker in a separate step.
+      - run:
+          name: Check newsfragment.
+          command: |
+            tox -r -e newsfragment
+      - run:
+          name: Check manifest-checker.
+          command: |
+            tox -r -e manifest-checker
+      - run:
+          name: Check pyflakes.
+          command: |
+            tox -r -e pyflakes
+
+
+  #
+  # Documentation tests are slow so we execute them in a separate job.
+  #
+  documentation:
```

Comment:
Why build documentation in CircleCI?  Won't that just burn through our allotted time faster?

Reply:
This is for Linux ... not the macOS and on CIrcle-CI we get 4 concurent builds.. and on Travis we get 5 AFAIK.

So since we have 4 "worked" available on Circle-CI I was thinking to use them here to reduce the load on Travis-CI.

Does that make sense?

I will try to create a documentation explaining how we run the automated tests.